doc/source/releases/17.01_announce.rst

```diff
@@ -10,17 +10,20 @@ Highlights
 
 **Conda auto initialization is enabled by default**
   After updating to 17.01 and upon starting Galaxy the Conda package manager will be installed automatically
-  within the Galaxy's workspace. This will enable your Galaxy to install tool dependencies using Conda
-  e.g. scientific `BioConda <https://bioconda.github.io/>`__ channel.
+  within the Galaxy's workspace. This will enable your Galaxy to install tool dependencies from Conda channels
+  such as `BioConda <https://bioconda.github.io/>`__. Such channels are already being used for new and updated tools by IUC
+  and we suggest adopting this tool dependency resolution method to all tool developers instead of Tool Shed package
+  recipes - which are now considered deprecated.
+  Please see the details at the admin `documentation <https://docs.galaxyproject.org/en/master/admin/conda_faq.html>`__
 
 **New interface for user preferences**
   User preferences menu has been reworked for clarity and consistency.
   Thanks to `@anuprulez <https://github.com/anuprulez>`__ and `@bgruening <https://github.com/bgruening>`__.
   Implemented in `Pull Request 3118`_.
 
-**Support for gzipped fastq formats**
-  You can now upload compressed files without being unpacked. Existing tools will work as before and future
-  tools will be able to consume archives and save space in your quota.
+**Support for compressed FASTQ formats**
+  You can now upload compressed (with gzip or bzip2) FASTQ files without them being unpacked automatically (as it was until now).
```

Comment:
Should we also specify that this applies only if you explicitly assign the corresponding datatype during file upload (sniffing disabled by default)?